### PR TITLE
rewrite README's introduction and leave placeholders

### PR DESCRIPTION
**Pull request by antonioalmeida**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-## Product View
+## Getting Started
 
+- [Features](#features)
+- [Setup Project](#setup-project)
+- [Architecture Overview](#architecture-overview)
+
+## Features
+
+[I WOULD REWRITE THIS]
```
Comment (diogotorres97) [inline README.md]: Maybe a bullet list with hfres

Comment (antonioalmeida) [reply README.md]: Agreed, but I would also write a general text on how/why each component is used. If you write a small draft, I can rewrite it

Comment (diogotorres97) [reply README.md]: **Main Features**
- [Express](https://expressjs.com) - Express is a minimal and flexible Node.js web application framework that provides a robust set of features to develop an api. 

- [Passport](http://www.passportjs.org) with [JWT](http://www.passportjs.org/packages/passport-jwt) - Passport is an authentication middleware for Node.js. We use it with JWT module that allows authentication endpoints using a JSON web token.

- [Postgres](https://www.postgresql.org) with [sequelize](https://sequelize.org) - PostgreSQL is an open source object-relational database system that has a strong reputation for reliability, feature robustness, and performance. We use it with Sequelize that is a promise-based Node.js ORM and it features solid transaction support, relations, eager and lazy loading, read replication and more.

- [Pgadmin](https://www.pgadmin.org) -pgAdmin is an Open Source management tool for Postgres that provides a powerful graphical interface that simplifies the creation, maintenance and use of database objects.

- [Docker](https://www.docker.com/get-started) - Docker containers wrap up software and its dependencies into a standardized unit for software development that includes everything it needs to run: code, runtime, system tools and libraries. This guarantees that your application will always run the same and makes collaboration as simple as sharing a container image.

- [Docker-Compose](https://docs.docker.com/compose) - Compose is a tool for defining and running multi-container Docker applications. With Compose we run a server API, the posgreSQL database and the pgadmin to interact with the database.

**Dev Features**

- [Chai](https://www.chaijs.com) - Chai is a BDD / TDD assertion library for node that assist in the construction of assertios in the tests developed.

- [Eslint](https://eslint.org) with [airbnb style](https://github.com/airbnb/javascript) - ESlint is a pluggable linting utility for JavaScript that with airbnb style provides a better readability to your code.

- [mocha](https://mochajs.org) -  Mocha is a test framework running on Node.js that allows asynchronous testing.

- [nodemon](https://nodemon.io) - Nodemon is a utility that will monitor for any changes in your source and automatically restart your server. Perfect for development.

- [nyc](https://istanbul.js.org) - NYC is a test coverage tool that track how well your unit-tests exercise your codebase.

Comment (diogotorres97) [reply README.md]: @antonioalmeida What do you think?

Comment (antonioalmeida) [reply README.md]: @diogotorres97 just rewrote, based on your suggestions

**Pull request merged**